omnigibson/systems/micro_particle_system.py

```diff
@@ -1528,7 +1534,7 @@ def cm_create_particle_template(cls):
 )
 
 
-class ClothSystem(MicroParticleSystem):
+class cloth(MicroParticleSystem):
```

Comment:
super nit, but it's a bit weird to have lower case classes. Could we maybe have simple logic that does the following:
- takes the first letter and capitalizes it. 
- takes any underscore, removes it, and then capitalizes the following letter

Just to maintain camelcasing for our classes :p

Reply:
done!